sspl_test/coverage/start_sspl_coverage.sh

```diff
@@ -43,8 +43,9 @@ copy_lines 10 24
 curr_line=`grep -n "#DO NOT EDIT: Marker comment to dynamically add signal handler for SIGUSR1 to generate code coverage report" $target_dir/sspl_ll_d_coverage | cut -d : -f1`
 copy_lines 25 25
 
-echo "Changing the soft link and adding permission for /tmp/sspl/ folder"
-ln -sf $target_dir/sspl_ll_d_coverage /usr/bin/sspl_ll_d
+echo "Changing existing sspl_ll_d file and adding permission for /tmp/sspl/ folder"
+sudo mv $target_dir/sspl_ll_d $target_dir/sspl_ll_d.back
+sudo mv $target_dir/sspl_ll_d_coverage $target_dir/sspl_ll_d
```

Comment:
why not soft-linking like existing way?

Reply:
Where to make a soft link of sspl_ll_d? In bin?
As you said in the last commit that this file should not directly accessible to the user.
https://github.com/Seagate/cortx-monitor/pull/304#discussion_r572871224

Reply:
No, i meant
> ln -sf $target_dir/sspl_ll_d_coverage $target_dir/sspl_ll_d

but anyway, nothing big achieved with this , so your way is fine